examples/carousel/carousel-2-tablist.html

```diff
@@ -11,42 +11,47 @@
     <script src="../js/highlight.pack.js"></script>
     <script src="../js/app.js"></script>
 
-    <!--  js and css for this example. -->
-    <script type="text/javascript">
-      var stylesheet = 'css/carousel-tablist.css';
-      if (location.href.indexOf('moreaccessible') > 0) {
-        console.log('loaded more accessible');
-        stylesheet = 'css/carousel-tablist-more-accessible.css';
-      }
-
-      document.write('<link rel="stylesheet" type="text/css" href="' + stylesheet + '">')
-
-      window.addEventListener('load', function () {
-        if (location.href.indexOf('moreaccessible') > 0) {
-          document.getElementById('id_link_more_accessible').setAttribute('aria-current', 'page');
+    <link rel="stylesheet" href="css/carousel-2-tablist.css">
+    <script src="js/carousel-tablist.js" type="text/javascript"></script>
+
+    <script>
+
+      window.addEventListener('load', function() {
+        var tablists = document.querySelectorAll('.carousel-tablist');
+
+        for (var i = 0; i < tablists.length; i++) {
+          var tablist = tablists[i].querySelector('[role=tablist]');
+          var items   = tablists[i].querySelector('.carousel-items');
+          width1 = tablist.getBoundingClientRect().width;
+          width2 = items.getBoundingClientRect().width;
+          tablist.style.left = (((width2-width1)/2) - 30) +'px';
         }
-        else {
-          document.getElementById('id_link_less_accessible').setAttribute('aria-current', 'page');
+      });
+
+      window.addEventListener('resize', function () {
+        var tablists = document.querySelectorAll('.carousel-tablist');
+
+        for (var i = 0; i < tablists.length; i++) {
+          var tablist = tablists[i].querySelector('[role=tablist]');
+          var items   = tablists[i].querySelector('.carousel-items');
+          width1 = tablist.getBoundingClientRect().width;
+          width2 = items.getBoundingClientRect().width;
+          tablist.style.left = (((width2-width1)/2) - 30) +'px';
         }
       });
 
     </script>
 
-    <script src="js/carousel-tablist.js" type="text/javascript"></script>
-
-  </head>
-  <body>
+  </head>  <body>
     <nav aria-label="Related Links" class="feedback">
       <ul>
         <li><a href="../../#browser_and_AT_support">Browser and Assistive Technology Support</a></li>
         <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
         <li><a href="https://github.com/w3c/aria-practices/projects/10">Related Issues</a></li>
         <li><a href="../../#carousel">Design Pattern</a></li>
       </ul>
-    </nav>
-    <main>
-      <h1>Example of Image Carousel with Automatically Rotating Images using a Tablist</h1>
-        <p>
+    </na
+    centerTablist(controls, items);v
```

Comment:
This looks like some javascript was accidentally pasted into the middle of the `</nav>` tag.

Reply:
@carmacleod 

Fixed, Carolyn thank you for all you feedback